### PR TITLE
Fixed the player only playing at about half speed.

### PR DESCRIPTION
**Pull request by Wcubed**
It turned out there was a signal loop between the Player and the horizontal slider.
When playing logs, the Player would emit the `positionChanged()` signal.
This would get picked up by the slider, which would set it's position accordingly.
The slider on it's turn would emit the `valueChanged()` signal.
This triggers the `seekFrame()` function of the mainwindow. Which then stops and starts the player.

This meant that the player was stopped and started for every frame it played, making it always late.
I fixed it by using the `sliderReleased()` signal instead of `valueChanged()`.

### CODE REVIEW

File: src/common/network.cpp [hunk 1]
```diff
@@ -57,8 +59,21 @@ void Network::connect()
 
     m_socket = new MulticastSocket(this);
     QObject::connect(m_socket, SIGNAL(readyRead()), SLOT(readData()));
-    m_socket->bind(m_port, QUdpSocket::ShareAddress  | QUdpSocket::ReuseAddressHint);
-    m_socket->joinMulticastGroup(m_groupAddress);
+
+    //m_socket->bind(m_port, QUdpSocket::ShareAddress  | QUdpSocket::ReuseAddressHint);
```
Comment (joydeep-b) [inline src/common/network.cpp]: The old commented out code should be removed.

File: src/common/network.cpp [hunk 2]
```diff
@@ -57,8 +59,21 @@ void Network::connect()
...
+
+    if(!m_socket->bind(m_port, QUdpSocket::ShareAddress | QUdpSocket::ReuseAddressHint)) {
+        std::cerr << "Unable to bind UDP socket on port " << m_port << ". "
+             << m_socket->errorString().toStdString() << '.' << endl;
+    }
+
+    //m_socket->joinMulticastGroup(m_groupAddress);
```
Comment (joydeep-b) [inline src/common/network.cpp]: Should be removed as well.

**Review by joydeep-b — COMMENTED** [review]
Commented out old code should be removed.

**Comment by Wcubed** [comment]
Good point, completely forgot to do that :)

**Comment by g3force** [comment]
I can not reproduce the original issue (running at half speed), but the changes look fine to be.
I also verified that the log player still builds and runs as expected.

**Comment by g3force** [comment]
I tried fixing the merge conflict with Github, but it does not seem to work. It looks like I actually managed to commit into the RoboTeamTwente repository.

**Comment by g3force** [comment]
@Wcubed it might be best if you rebase your master onto the current RoboCup-SSL master first.

**Comment by g3force** [comment]
see #5

**Pull request closed without merging**